frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -358,7 +360,7 @@ export const PersistentVolumeClaimsList = (props) => {
 
 export const PersistentVolumeClaimsPage = (props) => {
   const { t } = useTranslation();
-  const createPropExtensions = useExtensions(isPVCCreateProp);
+  const [createPropExtensions] = useResolvedExtensions(isPVCCreateProp);
```

Comment:
Why the change to `useResolvedExtensions`? Doesn't look like this requires resolving.